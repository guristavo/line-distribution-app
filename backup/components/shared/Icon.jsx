import React from 'react';
import PropTypes from 'prop-types';

import icons from '../icons';

export const ICONS_LIST = [
  'default',
  'archive',
  'box-checked',
  'box-invalid',
  'box-unchecked',
  'chevron-down',
  'chevron-up',
  'clock',
  'connect',
  'edit',
  'heart',
  'hide',
  'loading',
  'lyrics',
  'lyrics-connected',
  'no',
  'number',
  'number-1',
  'number-2',
  'number-3',
  'number-4',
  'number-5',
  'official',
  'plug',
  'plug-available',
  'plug-connected',
  'plug-linked',
  'reset',
  'results',
  'save',
  'sign-in',
  'sign-out',
  'stop',
  'trash',
  'used',
  'video',
  'view',
  'yes',
];

const Icon = ({ type, size = 'small', styles = {} }) => {
  let icon;

  switch (type) {
    case 'archive':
      icon = icons.archive;
      break;

    case 'box-checked':
      icon = icons.boxChecked;
      break;

    case 'box-invalid':
      icon = icons.boxInvalid;
      break;

    case 'box-unchecked':
      icon = icons.boxUnchecked;
      break;

    case 'chevron-down':
      icon = icons.chevronDown;
      break;

    case 'chevron-up':
      icon = icons.chevronUp;
      break;

    case 'clock':
      icon = icons.clock;
      break;

    case 'connect':
      icon = icons.connect;
      break;

    case 'edit':
      icon = icons.edit;
      break;

    case 'heart':
      icon = icons.heart;
      break;

    case 'hide':
      icon = icons.hide;
      break;

    case 'loading':
      icon = icons.loading;
      break;

    case 'lyrics':
      icon = icons.lyrics;
      break;

    case 'lyrics-connected':
      icon = icons.lyricsConnected;
      break;

    case 'no':
      icon = icons.no;
      break;

    case 'number':
      icon = icons.number;
      break;

    case 'number-1':
      icon = icons.number1;
      break;

    case 'number-2':
      icon = icons.number2;
      break;

    case 'number-3':
      icon = icons.number3;
      break;

    case 'number-4':
      icon = icons.number4;
      break;

    case 'number-5':
      icon = icons.number5;
      break;

    case 'official':
      icon = icons.official;
      break;

    case 'plug':
      icon = icons.plug;
      break;

    case 'plug-available':
      icon = icons.plugAvailable;
      break;

    case 'plug-connected':
      icon = icons.plugConnected;
      break;

    case 'plug-linked':
      icon = icons.plugLinked;
      break;

    case 'reset':
      icon = icons.reset;
      break;

    case 'results':
      icon = icons.results;
      break;

    case 'save':
      icon = icons.save;
      break;

    case 'sign-in':
      icon = icons.signIn;
      break;

    case 'sign-out':
      icon = icons.signOut;
      break;

    case 'stop':
      icon = icons.stop;
      break;

    case 'trash':
      icon = icons.trash;
      break;

    case 'used':
      icon = icons.used;
      break;

    case 'video':
      icon = icons.video;
      break;

    case 'view':
      icon = icons.view;
      break;

    case 'yes':
      icon = icons.yes;
      break;

    default:
      icon = icons.default;
  }

  const mergeStyles = (...args) => Object.assign({}, ...args);

  const style = {};

  return (
    <span className={`icon icon-${size}`}>
      <svg
        viewBox="0 0 100 100"
        preserveAspectRatio="xMidYMid meet"
        style={mergeStyles(
          styles,
          style // This lets the parent pass custom styles
        )}
      >
        {icon}
      </svg>
    </span>
  );
};

Icon.propTypes = {
  type: PropTypes.string,
  size: PropTypes.oneOf([
    'small',
    'small-inline',
    'medium',
    'medium-inline',
    'large',
    'x-large',
  ]),
  styles: PropTypes.object,
};

Icon.defaultProps = {
  type: 'default',
  size: 'small',
  styles: {},
};

export default Icon;
