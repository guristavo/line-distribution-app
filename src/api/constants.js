export const ALTERNATIVE_COLOR_LIST = {
  1: [31, 4, 17],
  2: [32, 5, 18],
  3: [33, 6, 19],
  4: [1, 7, 20],
  5: [2, 8, 21],
  6: [3, 9, 22],
  7: [4, 10, 23],
  8: [5, 11, 24],
  9: [6, 12, 25],
  10: [7, 13, 26],
  11: [8, 14, 27],
  12: [9, 15, 28],
  13: [10, 16, 29],
  14: [11, 17, 30],
  15: [12, 18, 31],
  16: [13, 19, 32],
  17: [14, 20, 33],
  18: [15, 21, 1],
  19: [16, 22, 2],
  20: [17, 23, 3],
  21: [18, 24, 4],
  22: [19, 25, 5],
  23: [20, 26, 6],
  24: [21, 27, 7],
  25: [22, 28, 8],
  26: [23, 29, 9],
  27: [24, 30, 10],
  28: [25, 31, 11],
  29: [26, 32, 12],
  30: [27, 33, 13],
  31: [28, 1, 14],
  32: [29, 2, 15],
  33: [30, 3, 16],
  34: [3, 8, 13, 26, 32],
  35: [4, 7, 12, 27],
  36: [6, 9, 16, 22, 31],
};
