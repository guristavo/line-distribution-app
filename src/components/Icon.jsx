import React from 'react';
import PropTypes from 'prop-types';

import icons from './icons';

export const ICONS_LIST = ['default', 'archive', 'box-checked', 'box-invalid', 'box-checked', 'clock', 'edit', 'heart', 'hide', 'loading', 'no', 'official', 'reset', 'results', 'save', 'sign-in', 'sign-out', 'stop', 'trash', 'used', 'view', 'yes'];

const Icon = ({ type, size = 'small', styles = {} }) => {
  let icon;

  switch (type) {
    case 'archive':
      icon = icons.archive;
      break;

    case 'box-checked':
      icon = icons.boxChecked;
      break;

    case 'box-invalid':
      icon = icons.boxInvalid;
      break;

    case 'box-unchecked':
      icon = icons.boxUnchecked;
      break;

    case 'clock':
      icon = icons.clock;
      break;

    case 'edit':
      icon = icons.edit;
      break;

    case 'heart':
      icon = icons.heart;
      break;

    case 'hide':
      icon = icons.hide;
      break;

    case 'loading':
      icon = icons.loading;
      break;

    case 'no':
      icon = icons.no;
      break;

    case 'official':
      icon = icons.official;
      break;

    case 'reset':
      icon = icons.reset;
      break;

    case 'results':
      icon = icons.results;
      break;

    case 'save':
      icon = icons.save;
      break;

    case 'sign-in':
      icon = icons.signIn;
      break;

    case 'sign-out':
      icon = icons.signOut;
      break;

    case 'stop':
      icon = icons.stop;
      break;

    case 'trash':
      icon = icons.trash;
      break;

    case 'used':
      icon = icons.used;
      break;

    case 'view':
      icon = icons.view;
      break;

    case 'yes':
      icon = icons.yes;
      break;

    default:
      icon = icons.default;
  }

  const mergeStyles = (...args) => Object.assign({}, ...args);

  const style = {};

  return (
    <span className={`icon icon-${size}`}>
      <svg
        viewBox="0 0 100 100"
        preserveAspectRatio="xMidYMid meet"
        style={mergeStyles(
          styles,
          style, // This lets the parent pass custom styles
        )}
      >
        { icon }
      </svg>
    </span>

  );
};

Icon.propTypes = {
  type: PropTypes.string, // eslint-disable-line
  size: PropTypes.oneOf(['small', 'medium', 'large', 'x-large']), // eslint-disable-line
  styles: PropTypes.object, // eslint-disable-line
};

export default Icon;
