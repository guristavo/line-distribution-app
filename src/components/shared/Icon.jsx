import React from 'react';
import PropTypes from 'prop-types';

import icons from '../icons';

export const ICONS_LIST = [
  'default',
  'archive',
  'box-checked',
  'box-invalid',
  'box-unchecked',
  'chevron-down',
  'chevron-up',
  'clock',
  'connect',
  'edit',
  'heart',
  'hide',
  'loading',
  'lyrics',
  'lyrics-connected',
  'no',
  'official',
  'plug',
  'plug-connected',
  'reset',
  'results',
  'save',
  'sign-in',
  'sign-out',
  'stop',
  'trash',
  'used',
  'video',
  'view',
  'yes',
];

const Icon = ({ type, size = 'small', styles = {} }) => {
  let icon;

  switch (type) {
    case 'archive':
      icon = icons.archive;
      break;

    case 'box-checked':
      icon = icons.boxChecked;
      break;

    case 'box-invalid':
      icon = icons.boxInvalid;
      break;

    case 'box-unchecked':
      icon = icons.boxUnchecked;
      break;

    case 'chevron-down':
      icon = icons.chevronDown;
      break;

    case 'chevron-up':
      icon = icons.chevronUp;
      break;

    case 'clock':
      icon = icons.clock;
      break;

    case 'connect':
      icon = icons.connect;
      break;

    case 'edit':
      icon = icons.edit;
      break;

    case 'heart':
      icon = icons.heart;
      break;

    case 'hide':
      icon = icons.hide;
      break;

    case 'loading':
      icon = icons.loading;
      break;

    case 'lyrics':
      icon = icons.lyrics;
      break;

    case 'lyrics-connected':
      icon = icons.lyricsConnected;
      break;

    case 'no':
      icon = icons.no;
      break;

    case 'official':
      icon = icons.official;
      break;

    case 'plug':
      icon = icons.plug;
      break;

    case 'plug-connected':
      icon = icons.plugConnected;
      break;

    case 'reset':
      icon = icons.reset;
      break;

    case 'results':
      icon = icons.results;
      break;

    case 'save':
      icon = icons.save;
      break;

    case 'sign-in':
      icon = icons.signIn;
      break;

    case 'sign-out':
      icon = icons.signOut;
      break;

    case 'stop':
      icon = icons.stop;
      break;

    case 'trash':
      icon = icons.trash;
      break;

    case 'used':
      icon = icons.used;
      break;

    case 'video':
      icon = icons.video;
      break;

    case 'view':
      icon = icons.view;
      break;

    case 'yes':
      icon = icons.yes;
      break;

    default:
      icon = icons.default;
  }

  const mergeStyles = (...args) => Object.assign({}, ...args);

  const style = {};

  return (
    <span className={`icon icon-${size}`}>
      <svg
        viewBox="0 0 100 100"
        preserveAspectRatio="xMidYMid meet"
        style={mergeStyles(
          styles,
          style // This lets the parent pass custom styles
        )}
      >
        {icon}
      </svg>
    </span>
  );
};

Icon.propTypes = {
  type: PropTypes.string,
  size: PropTypes.oneOf([
    'small',
    'small-inline',
    'medium',
    'medium-inline',
    'large',
    'x-large',
  ]),
  styles: PropTypes.object,
};

Icon.defaultProps = {
  type: 'default',
  size: 'small',
  styles: {},
};

export default Icon;
