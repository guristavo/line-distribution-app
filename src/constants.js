import ARTISTS_DB from './database';

export const COLORS_LIST = ['blood', 'redOrange', 'peach', 'brown', 'orange', 'dirt', 'lightBrown', 'sand', 'gold', 'tan', 'yellow', 'olive', 'pee', 'yellowGreen', 'green', 'darkGreen', 'forest', 'lime', 'teal', 'cyan', 'turquoise', 'navy', 'blue', 'purple', 'violet', 'rose', 'plum', 'pink', 'redViolet', 'hotPink', 'wine', 'salmon', 'red', 'lightGrey', 'grey', 'darkGrey'];

export const KEYS = {
    49: {
        keyCode: 49,
        keyFace: '1',
        id: 0
    },
    50: {
        keyCode: 50,
        keyFace: '2',
        id: 1
    },
    51: {
        keyCode: 51,
        keyFace: '3',
        id: 2
    },
    52: {
        keyCode: 52,
        keyFace: '4',
        id: 3
    },
    53: {
        keyCode: 53,
        keyFace: '5',
        id: 4
    },
    54: {
        keyCode: 54,
        keyFace: '6',
        id: 5
    },
    55: {
        keyCode: 55,
        keyFace: '7',
        id: 6
    },
    56: {
        keyCode: 56,
        keyFace: '8',
        id: 7
    },
    57: {
        keyCode: 57,
        keyFace: '9',
        id: 8
    },
    48: {
        keyCode: 48,
        keyFace: '0',
        id: 9
    },
    81: {
        keyCode: 81,
        keyFace: 'Q',
        id: 10
    },
    87: {
        keyCode: 87,
        keyFace: 'W',
        id: 11
    },
    69: {
        keyCode: 69,
        keyFace: 'E',
        id: 12
    },
    82: {
        keyCode: 82,
        keyFace: 'R',
        id: 13
    },
    84: {
        keyCode: 84,
        keyFace: 'T',
        id: 14
    },
    89: {
        keyCode: 89,
        keyFace: 'Y',
        id: 15
    },
    85: {
        keyCode: 85,
        keyFace: 'U',
        id: 16
    },
    73: {
        keyCode: 73,
        keyFace: 'I',
        id: 17
    },
    79: {
        keyCode: 79,
        keyFace: 'O',
        id: 18
    },
    80: {
        keyCode: 80,
        keyFace: 'P',
        id: 19
    },
    65: {
        keyCode: 65,
        keyFace: 'A',
        id: 20
    },
    83: {
        keyCode: 83,
        keyFace: 'S',
        id: 21
    },
    68: {
        keyCode: 68,
        keyFace: 'D',
        id: 22
    },
    70: {
        keyCode: 70,
        keyFace: 'F',
        id: 23
    },
    71: {
        keyCode: 71,
        keyFace: 'G',
        id: 24
    },
};

export const KEYCODE_LIST = [49, 50, 51, 52, 53, 54, 55, 56, 57, 48, 81, 87, 69, 82, 84, 89, 85, 73, 79, 80, 65, 83, 68, 70, 71];

export const KEY_LIST = [1, 2, 3, 4, 5, 6, 7, 8, 9, 0, 'Q', 'W', 'E', 'R', 'T', 'Y', 'U', 'I', 'O', 'P', 'A', 'S', 'D', 'F', 'G'];

export const ARTISTS = ARTISTS_DB;

export const ARTITST_PLACEHOLDER = {
    bandName: 'No Artists Selected',
    members: [''],
    colors: ['']
};
