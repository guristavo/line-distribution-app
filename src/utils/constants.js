const COLORS = {
  col000001: 'red',
  col000002: 'blood',
  col000003: 'coral',
  col000004: 'peach',
  col000005: 'brown',
  col000006: 'orange',
  col000007: 'gold',
  col000008: 'yellow',
  col000009: 'olive',
  col000010: 'lemon',
  col000011: 'lime',
  col000012: 'green',
  col000013: 'basil',
  col000014: 'forest',
  col000015: 'jade',
  col000016: 'teal',
  col000017: 'cyan',
  col000018: 'turquoise',
  col000019: 'navy',
  col000020: 'blue',
  col000021: 'sky',
  col000022: 'purple',
  col000023: 'violet',
  col000024: 'rose',
  col000025: 'mauve',
  col000026: 'magenta',
  col000027: 'jam',
  col000028: 'pink',
  col000029: 'burgundy',
  col000030: 'salmon',
};

const DEFAULT_COLORS = {
  default: '#000000',
  black: '#000000',
  blue: '#598ffd',
  gray: '#707070',
  green: '#49d156',
  orange: '#fea245',
  pink: '#fb9cf0',
  purple: '#9e8ef0',
  red: '#d14949',
  teal: '#34ac9e',
  white: 'FFFFFF',
  yellow: '#ffd700',
};

const FLAGS_LIST = {
  AMERICAN: 'usa',
  BRAZILIAN: 'brazil',
  BRITISH: 'uk',
  CANADIAN: 'canada',
  CHINESE: 'china',
  JAPANESE: 'japan',
  KOREAN: 'korea',
  PLANTIAN: 'planted',
  TAIWANESE: 'taiwan',
  THAI: 'thailand',
  VIETNAMISE: 'vietnam',
};

const GENRES_DB = {
  'C-Pop': 'CPOP',
  'J-Pop': 'JPOP',
  'K-Pop': 'KPOP',
  Other: 'OTHER',
  Pop: 'POP',
};

const POSITIONS_LIST = [
  'MAIN_VOCALIST',
  'MAIN_RAPPER',
  'MAIN_DANCER',
  'LEAD_VOCALIST',
  'LEAD_RAPPER',
  'LEAD_DANCER',
  'VOCALIST',
  'RAPPER',
  'DANCER',
  'LEADER',
  'CENTER',
  'VISUAL',
];

const PROFILE_PICTURE_URL = '/images/profiles/';

const GROUPS_PICTURE_URL = '/images/groups/';

const SYNC_KEY_LIST = {
  1: true,
  2: true,
  3: true,
  4: true,
  5: true,
};

const SYNC_KEY_COLOR_LIST = {
  1: '1',
  2: '8',
  3: '12',
  4: '20',
  5: '26',
};

const YOUTUBE_URLS = ['cudssvDuOpc'];

export default {
  COLORS,
  DEFAULT_COLORS,
  FLAGS_LIST,
  GENRES_DB,
  GROUPS_PICTURE_URL,
  POSITIONS_LIST,
  PROFILE_PICTURE_URL,
  SYNC_KEY_LIST,
  SYNC_KEY_COLOR_LIST,
  YOUTUBE_URLS,
};
