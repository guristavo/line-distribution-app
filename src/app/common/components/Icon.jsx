import React from 'react';
import PropTypes from 'prop-types';

import { DEFAULT_COLORS } from '../../../utils/constants';
import ICONS from '../../../utils/icons';

const Icon = ({ type, size, color, inline }) => {
  const margin = inline ? '0 3px' : 0;

  const styles = {
    svg: {
      display: 'inline-block',
      verticalAlign: 'middle',
      margin,
    },
    path: {
      fill: DEFAULT_COLORS[color],
    },
  };

  return (
    <svg
      style={styles.svg}
      width={`${size}px`}
      height={`${size}px`}
      viewBox="0 0 100 100"
    >
      <path style={styles.path} d={ICONS[type]} className={`icon-${type}`} />
    </svg>
  );
};

Icon.propTypes = {
  type: PropTypes.string,
  size: PropTypes.oneOfType([PropTypes.number, PropTypes.string]),
  color: PropTypes.string,
  inline: PropTypes.bool,
};

Icon.defaultProps = {
  type: 'default',
  size: 12,
  color: 'default',
  inline: false,
};

export default Icon;
