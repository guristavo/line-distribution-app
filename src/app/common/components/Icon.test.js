import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Icon from './Icon';

import constants from '../../../utils/constants';
import ICONS from '../../../utils/icons';

describe('common/Icon', () => {
  afterEach(cleanup);

  it('renders', () => {
    const { asFragment } = render(<Icon />);
    expect(asFragment()).toMatchSnapshot();
  });

  const iconsList = Object.keys(ICONS);
  for (let i = 0; i < iconsList.length; i++) {
    const icon = iconsList[i];
    it('accepts a valid type from the ICONS svg object', () => {
      const { getByTitle } = render(<Icon type={icon} />);
      expect(getByTitle(icon).firstChild.classList.value.trim()).toBe(
        `icon-${icon}`
      );
    });
  }

  const colorsList = Object.entries(constants.DEFAULT_COLORS);
  for (let i = 0; i < colorsList.length; i++) {
    const { color, hex } = colorsList[i];
    it('accepts a color argument from the colors constant', () => {
      const { getByTestId } = render(<Icon color={color} />);
      expect(getByTestId('icon-path')).toHaveAttribute('style', hex);
    });
  }

  it('accepts a size argument', () => {
    const { getByTitle } = render(<Icon size="10" />);
    expect(getByTitle('default')).toHaveAttribute('width', '10px');
    expect(getByTitle('default')).toHaveAttribute('height', '10px');
  });

  it('accepts a title argument', () => {
    const { getByTitle } = render(<Icon title="Custom Title" />);
    expect(getByTitle('Custom Title')).toBeInTheDocument();
  });

  it('accepts an inline flag', () => {
    const { getByTitle } = render(<Icon inline />);
    expect(getByTitle('default')).toHaveAttribute(
      'style',
      'display: inline-block; vertical-align: middle; margin: 0px 3px; transform: translateY(-1px);'
    );
  });
});
