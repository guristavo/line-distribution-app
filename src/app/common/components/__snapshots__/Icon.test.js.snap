// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`common/Icon renders 1`] = `
<DocumentFragment>
  <svg
    height="12px"
    style="display: inline-block; vertical-align: middle; margin: 0px; transform: translateY(-1px);"
    title="default"
    viewBox="0 0 100 100"
    width="12px"
  >
    <path
      class="icon-default"
      d="M81.2,12.5c3.8,0,6.2,2.5,6.2,6.2v62.5c0,3.8-2.5,6.2-6.2,6.2H18.8c-3.7,0-6.2-2.5-6.2-6.2V18.8 c0-3.7,2.5-6.2,6.2-6.2H81.2 M81.2,0H18.8C8.1,0,0,8.1,0,18.8v62.5C0,91.9,8.1,100,18.8,100h62.5c10.6,0,18.8-8.1,18.8-18.8V18.8 C100,8.1,91.9,0,81.2,0L81.2,0z M35.3,31.1v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C33.3,35.7,35.3,33.6,35.3,31.1z M58.3,31.1v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C56.3,35.7,58.3,33.6,58.3,31.1z M81.3,31.1v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C79.3,35.7,81.3,33.6,81.3,31.1z M35.3,53.8v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C33.3,58.3,35.3,56.3,35.3,53.8z M58.3,53.8v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C56.3,58.3,58.3,56.3,58.3,53.8z M81.3,53.8v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C79.3,58.3,81.3,56.3,81.3,53.8z M35.3,76.5v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C33.3,81,35.3,79,35.3,76.5z M58.3,76.5v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C56.3,81,58.3,79,58.3,76.5z M81.3,76.5v-7.6c0-2.5-2-4.5-4.5-4.5h-7.6c-2.5,0-4.5,2-4.5,4.5v7.6c0,2.5,2,4.5,4.5,4.5h7.6 C79.3,81,81.3,79,81.3,76.5z"
      data-testid="icon-path"
      style="fill: #000000;"
    >
      <title>
        default
      </title>
    </path>
  </svg>
</DocumentFragment>
`;
